activities/models.py

```diff
@@ -4,37 +4,43 @@
 
 
 class Activity(models.Model):
-    name = models.CharField(max_length=50)
-    description = models.TextField(blank=True, default='n/a')
+    name = models.CharField(max_length=150)
+    description = models.TextField(blank=True)
     hide_activity = models.BooleanField(default=False)
-    hide_participants = models.BooleanField(default=False)
-    cost = models.CharField(max_length=150, default='n/a')
-    location = models.CharField(max_length=150, default='n/a')
-    start_date = models.DateTimeField(null=True,
-                                      blank=True,
-                                      verbose_name='Start date')
-    end_date = models.DateTimeField(null=True,
-                                    blank=True,
-                                    verbose_name='End date')
+    hide_participants = models.BooleanField(default=False,
+                                            help_text="Participants will be hidden for everyone except the organizers.")
+    cost = models.DecimalField(max_digits=8,
```

Comment:
Ik heb cost liever een Charfield, omdat je dan ook kunt bij schrijven per wat de kosten zijn: per persoon, maaltijd, dag, meerdere kosten (student/burger of één nacht/twee nachten).
Activiteiten zijn over het algemeen een beetje los in het handelen van kosten, dus het is handiger om dit gewoon communicatie tussen de penno en de commissie te laten.

Reply:
Is prima, ik zal het terugveranderen.